Lection 1/task3.js

```diff
@@ -0,0 +1,6 @@
+// Task3**. Напишите программу, которая принимает целое положительное число n (одно любое число от 1 до 9), и выводит сумму равную 
+// n + nn + nnn, где n не перемножаются, а конкатенируются
+const sum = (n) => {
```

Comment:
Please provide validation for n argument according to requirements:
- number 
- min 1
- max 9